synthesizer/src/program/instruction/operation/assert.rs

```diff
@@ -263,7 +265,6 @@ mod tests {
         cache: &mut HashMap<String, (ProvingKey<CurrentNetwork>, VerifyingKey<CurrentNetwork>)>,
     ) -> Result<(Stack<CurrentNetwork>, Vec<Operand<CurrentNetwork>>)> {
```

Comment:
Shouldn't `run` be outputted from this method if we're really abstracting it?

Reply:
Agreed, but we're not abstracting `sample_stack`in this PR.
That could be done in a follow up PR, which dedups `sample_stack` across all this tests in `operations`